script.js

```diff
@@ -0,0 +1,46 @@
+function table() {
+    let table = document.querySelector('.table');
```

Comment:
Використовуй `const` для змінних, які не перезаписуєш.

Reply:
використовуй 2 пробіли замість 4
```suggestion
  const table = document.querySelector('.table');
```